### PR TITLE
738 add missing request on FHIR bundle

### PR DESCRIPTION
**Pull request by leite08**
Ref. metriport/metriport-internal#738

### Dependencies

none

### Description

- add missing request on FHIR bundle after converting from CDA to FHIR
- increase IOPS alarm from 5,000 to 10,000 IOPS

### Release Plan

- ⚠️ This is pointing to `master`

### CODE REVIEW

File: infra/lib/api-stack.ts [hunk 1]
```diff
@@ -784,7 +784,7 @@ export class APIStack extends Stack {
 
     const writeIOPsMetric = dbCluster.metricVolumeWriteIOPs();
     const wIOPSAlarm = writeIOPsMetric.createAlarm(this, `${dbClusterName}VolumeWriteIOPsAlarm`, {
-      threshold: 5000, // IOPs per second
+      threshold: 10000, // IOPs per second
```
Comment (leite08) [inline infra/lib/api-stack.ts:787]: getting alarms with just one patient being processed... likely could increase even further

https://metriport.slack.com/archives/C04T256DQPQ/p1685576567720709

File: api/lambdas/sqs-to-converter/index.js [hunk 2]
```diff
@@ -147,6 +147,7 @@ export const handler = Sentry.AWSLambda.wrapHandler(async event => {
         await reportMemoryUsage();
         addExtensionToConversion(conversionResult, documentExtension);
         removePatientFromConversion(conversionResult);
+        addMissinRequests(conversionResult);
```
Comment (Goncharo) [inline api/lambdas/sqs-to-converter/index.js:150]: `addMissingRequests`

File: api/lambdas/sqs-to-converter/index.js [hunk 3]
```diff
@@ -204,6 +205,17 @@ function removePatientFromConversion(conversion) {
   if (pos >= 0) conversion.fhirResource.entry.splice(pos, 1);
 }
 
+function removePatientFromConversion(conversion) {
```
Comment (Goncharo) [inline api/lambdas/sqs-to-converter/index.js:208]: `addMissingRequests()`

**Review by Goncharo — CHANGES_REQUESTED** [review]

File: api/lambdas/sqs-to-converter/index.js [hunk 4]
```diff
@@ -204,6 +205,17 @@ function removePatientFromConversion(conversion) {
...
+  conversion.fhirResource.entry.forEach(e => {
+    if (!e.request) {
+      e.request = {
+        method: "PUT",
+        url: `${e.resource.resourceType}/${e.resource.id}}`,
+      };
+    }
+  });
```
Comment (leite08) [inline api/lambdas/sqs-to-converter/index.js:216]: Tested local

image

File: api/lambdas/sqs-to-converter/index.js (hunk 3 above)
Comment (leite08) [reply api/lambdas/sqs-to-converter/index.js]: 🕵️

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**